examples/popup.html

```diff
@@ -10,10 +10,8 @@
     <link rel="stylesheet" href="../resources/bootstrap/css/bootstrap-responsive.min.css" type="text/css">
     <style type="text/css">
       .ol-popup {
-        display: none;
```

Comment:
I haven't checked your branch, but do you get a flash of popup content if `display` is not `none` on load?